packages/actor-http-native/lib/Requester.ts

```diff
@@ -46,6 +57,17 @@ export default class Requester {
     return requestProxy;
   }
 
+  // Wrap headers into an header object type
+  public convertFetchHeadersToHash(headers: IncomingHttpHeaders): Headers {
```

Comment:
The method name is not really correct anymore, should be something like `convertRequestHeadersToFetchHeaders`